frontend/package.json

```diff
@@ -6,7 +6,7 @@
 		"dev": "vite --open",
 		"build": "tsc && vite build",
 		"format": "prettier --write .",
-		"lint": "eslint . --fix",
+		"lint": "eslint . --fix && stylelint '**/*.{css,scss}' --fix",
```

Comment:
Good shout on using concurrently!
Interestingly, if I change the command to `stylelint '**/*.css' --fix`, the linter claims it can't find any files which match that 🤔 not sure if I'm missing something obvious there.
I've change the file format to be .cjs to match the eslint one, but I'm thinking .mjs may be more appropriate for both?

Reply:
Currently we can't use mjs with eslint, as far as I can see, so we're stuck with cjs until eslint adopts its new config format officially (or we switch, which seems unnecessary at this stage).

Weird that just ".css" didn't find anything, something funky definitely going on there. I'll check it out and see...